crates/resolver/src/config.rs

```diff
@@ -991,7 +991,7 @@ impl Default for ResolverOpts {
     fn default() -> Self {
         Self {
             ndots: 1,
-            timeout: Duration::from_secs(5),
+            timeout: Duration::from_millis(100),
```

Comment:
I don't think this is a reasonable general timeout base on my previous comment, real world latencies. I think I'd be ok with this being lowered to 2 seconds, but 100ms is too aggressive IMO.